apps/reference/docs/guides/cli/testing.mdx

```diff
@@ -0,0 +1,68 @@
+---
+id: testing
+title: Testing Database
+description: How to test your schema changes locally in the database.
+---
+
+import Tabs from '@theme/Tabs'
+import TabItem from '@theme/TabItem'
+
+You can use the Supabase CLI to test your database. Make sure you have [installed the CLI](https://github.com/supabase/cli#getting-started) on your local machine and [linked](https://supabase.com/docs/reference/cli/usage#supabase-link) your Supabase project.
```

Comment:
```suggestion
You can use the Supabase CLI to test your database. To get started:

- [install the CLI](https://github.com/supabase/cli#getting-started) on your local machine 
- [link](/docs/reference/cli/usage#supabase-link) your Supabase project
```

Reply:
@sweatybridge perhaps we can move the installation instructions to the docs? (doesn't need to be in this PR, just in the future)

Reply:
Sure, I will link to overview which has installation instructions. https://supabase.com/docs/guides/cli